src/services/completions.ts

```diff
@@ -126,11 +126,18 @@ namespace ts.Completions {
 
         // Use a 'sortText' of 0' so that all symbol completion entries come before any other
         // entries (like JavaScript identifier entries).
+
+        let sortText = "0";
+
+        if (symbol.parent && symbol.parent.mergeId) {

```

Comment:
This will touch more than classes, e.g. namespaces.. also you should not use mergeId outside the checker.

Reply:
also please add tests.